src/Xamarin.Android.Tools.AndroidSdk/Sdks/AndroidSdkUnix.cs

```diff
@@ -221,10 +221,16 @@ void AddToList (string path)
 		//
 		private static string UnixConfigPath {
 			get {
-				var p   = AppDomain.CurrentDomain.GetData (GetUnixConfigDirOverrideName)?.ToString ();
+				var p = AppDomain.CurrentDomain.GetData (GetUnixConfigDirOverrideName)?.ToString ();
 				if (string.IsNullOrEmpty (p)) {
 					p   = Environment.GetFolderPath (Environment.SpecialFolder.ApplicationData);
-					p   = Path.Combine (p, "xbuild");
+					if (string.IsNullOrEmpty(p))
+					{
+						// SpecialFolder.ApplicationData returns nothing when running on NET6, so in that
+						// case, fallback to the well known place
+						p = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), ".config");
+					}
```

Comment:
If we still need this `if`, we normally use this code style:

```suggestion
					if (string.IsNullOrEmpty (p))
					{
						// SpecialFolder.ApplicationData returns nothing when running on NET6, so in that
						// case, fallback to the well known place
						p = Path.Combine (Environment.GetFolderPath (Environment.SpecialFolder.UserProfile), ".config");
					}
```

It's in the `.editorconfig`, maybe this setting isn't working?

https://github.com/xamarin/xamarin-android-tools/blob/0d55472777ebf1a38b72a87c5bd3f72ecafa4832/.editorconfig#L183

Reply:
Fixed in https://github.com/xamarin/xamarin-android-tools/pull/188/commits/58a1c3bb038751deca2ccf6ea0aa08cec6d5c4ac